curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646ce8bb4b35544d501c7184.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 646ce8bb4b35544d501c7184
+title: Step 20
+challengeType: 0
+dashedName: step-20
+---
+
+# --description--
+
+Inside your `.cat-ears` element, create two `div` elements with the classes `left-ear` and `right-ear` respectively.
```

Comment:
`left-ear` and `right-ear` class names are inconsistent with naming of other classes which always have `cat-` prefix.

Reply:
Thank you. I'll change it.